readme.md

```diff
@@ -42,6 +42,10 @@ gulp.task('default', () =>
 
 #### options
 
+gulp-mocha will pass any options defined directly to the `mocha` binary. That
+means you have every [command line option](http://mochajs.org/#usage) available
```

Comment:
That's not the point. Users don't want to use quotes in the keys. And they wouldn't even know camelCase is supported.